CHANGES.md

```diff
@@ -20,6 +20,7 @@
   + Use opt and fmt_opt to simplify formatting (#1150) (Guillaume Petiot)
   + Replace inplace formatting by dune staging for make fmt (#1151) (Guillaume Petiot)
   + Refactor code that interprets CLI options (#1127) (Jules Aguillon)
+    Fix --enable-outside-project, broken by this PR (#1156) (Jules Aguillon)
```

Comment:
You can just update the above line with `(#1127, #1156)`, I think it's enough.

Reply:
Agree.